tests/examples/hello/src/sum.cc

```diff
@@ -0,0 +1,34 @@
+#include "hello_world.h"
+#include "legate_library.h"
+
+namespace hello {
+
+class SumTask : public Task<SumTask, SUM> {
+public:
+  static void cpu_variant(legate::TaskContext &context) {
+    auto &input = context.inputs()[0];
+    auto input_shape = input.shape<1>(); // should be a 1-Dim array
+    auto in = input.read_accessor<float, 1>();
+
+    auto n = input_shape.volume();
+    float total = 0;
+    for (size_t i = 0; i < n; ++i) {
+      total += in[i];
+    }
+
+    using Reduce = Legion::SumReduction<float>;
+    auto sum = context.reductions()[0].reduce_accessor<Reduce, true, 1>();
```

Comment:
Suggest adding some comments to explain what's going on.

"The task launch as a whole will return a single value (`Store` of size 1) to the caller. However, each point task gets a separate `Store` of the same size as the result, to reduce into. This "local accumulator" will be initialized by the runtime, and all we need to do is call `.reduce()` to add our local contribution. After all point tasks return, the runtime will make sure to combine all their buffers into the single final result."